### DISCUSSION TITLE
Retries not showing in allure-report

### DISCUSSION THREAD

**Discussion by pwpatton** [Shelex/cypress-allure-plugin#137] [General]
Hello, I'm not sure if I missed an option or if this is a known issue. 
But I have retires set to 2 in cypress.json

``` 
 "retries": {
    "runMode": 2,
    "openMode": 0
  },
  ```
  
The retries appear to happen as you can see in the 3 screen shots in the overview tab

![image](https://user-images.githubusercontent.com/4737815/166297916-58fe4a1e-6c7b-470c-8a19-1ddd5509eb44.png)

But nothing appears in the retries tab

![image](https://user-images.githubusercontent.com/4737815/166297981-8804c977-baae-4929-a021-6ae5544e4be4.png)

Please advise

**Comment by Shelex** [comment]
I am not much using retries so I am not sure if it is handled properly. 
Latest result should be recorded just fine, but not sure if previous attempts are included.

Allure treats a "retry" the test with same name, so it is doable in general. 
The problem is if cypress fires proper events for mocha so we could track that.

I will convert this thread as an enhancement ticket as better investigation is required.

**Reply by pwpatton** [reply]
Thanks @Shelex,
 
Interestingly. I noticed content in the retries tab as a result of running a single test w/o first killing my cypress results (and then regenerating the report). I didn't try and replicate this but was thinking the retries tab could be about folks re-running failed tests?!

**Reply by Shelex** [reply]
Exactly. When allure-results contain several files with same test names - previous results will be treated as retries.
However plugin does not save results for previous attempts, attachments only.
I do not consider this feature as high priority, it may be tricky to implement due to events that cypress fire in mocha in case of retries.
However it makes sense to track it in terms of enhancement ticket, which I will address when will have time and inspiration :)

**Comment by Shelex** [comment]
Closing as it was released in 2.36.0
